libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSourceTest.kt

```diff
@@ -125,7 +125,7 @@ class MapboxNavigationViewportDataSourceTest {
 
     @Before
     fun setup() {
-        mockkStatic(DirectionsRoute::toNavigationRoute)
+        mockkStatic("com.mapbox.navigation.base.route.NavigationRouteEx")
```

Comment:
Does the function reference not work?

Reply:
it's not working anymore, when a function is overloaded the compiler cannot resolve which methods must be used